db/migrate/20170304172631_create_hero_pools.rb

```diff
@@ -0,0 +1,10 @@
+class CreateHeroPools < ActiveRecord::Migration[5.0]
+  def change
+    create_table :hero_pools do |t|
```

Comment:
Yes, thank you! Rob referred to them as "hero pools" initially which works well to describe the concept, but it wasn't quite right for a table name and I didn't think of anything better.